node-serverless-app/README.md

```diff
@@ -0,0 +1,29 @@
+# node-serverless-app
+
+This example illustrates a Lambda function instrumented with Honeycomb's [Beeline for Node](https://docs.honeycomb.io/getting-data-in/javascript/beeline-nodejs/).
+
+It contains examples of:
+
+- Baseline Beeline usage (not in an Express app)
+- Capture of custom metadata on Beeline-generated events
+- Definition of custom spans to augment traces
+- Continuing a propagated trace (if applicable), e.g. if the client initiated a trace via another Honeycomb Beeline
+- Error handling with a trace
+
+We've intentionally included the `node_modules` directory in hopes of simplifying dependencies.
```

Comment:
Err, this part isn't true in this repo, right?

Reply:
ha! thanks. it was true until i realized how large those directories would be, thanks for the catch(es)!